.github/bot/package.json

```
@@ -0,0 +1,37 @@
+{
+  "name": "auto-merge-eip",
+  "version": "1.0.0",
+  "description": "auto merge eip",
+  "main": "index.js",
+  "scripts": {
+    "test": "NODE_ENV=testing jest __tests__",
+    "build": "ncc build ./src/index.ts",
+    "watch": "ncc build ./src/index.ts --watch",
+    "it": "node -r dotenv/config dist/index.js",
+    "prettier-changed": "prettier --write"
+  },
+  "author": "",
+  "license": "ISC",
```

```suggestion
  "license": "CC0-1.0",
```
Everything in this repository should be CC0 licensed.